lib/rmt/cli/main.rb

```diff
@@ -59,11 +59,15 @@ def mirror
   desc 'export', _('Export commands for Offline Sync')
   subcommand 'export', RMT::CLI::Export
 
+  desc 'registrations', _('Commands for managing registered systems')
```

Comment:
Do you think it would be good to adjust the description text to `List Systems` (In case you change it to systems as @Sergeykot mentioned)? Then it follows the pattern of the other descriptions of `products` and `repos` and it looks more consistent.